app/src/ui/lib/install-cli.ts

```diff
@@ -1,11 +1,7 @@
-import * as Fs from 'fs'
+import * as FSE from 'fs-extra'
 import * as Path from 'path'
 
-const runas: (
-  command: string,
-  args: ReadonlyArray<string>,
-  options?: { admin: boolean }
-) => number = require('runas')
+import fsAdmin = require('fs-admin')
```

Comment:
Are we unable to do TS style import: `import * as fsAdmin from 'fs-admin`?

Reply:
I think this was just an oversight because the type declarations weren't setup correctly. Fixed in 4fc3a1717401f46971b5967f6b6b807b7f062149